### DISCUSSION TITLE
Dropdown Menu Dual Item

### DISCUSSION THREAD

**Discussion by Sashkan** [radix-ui/primitives#1438] [Ideas]
Is there a way to combine an `Item` and a `TriggerItem` inside of a dropdown menu?

According to the docs, pressing space/enter will open the submenu on a TriggerItem, and activate an Item.

What I'd like to do is to activate the focused item when pressing space/enter, and toggle the submenu when pressing the right arrow. Is it possible to get this kind of behavior natively?

**Comment by andy-hook** [comment]
Can I ask what your use case is? I'm struggling to see why you would need the item itself to be selectable at the same time as being parent to sub items.

**Reply by Sashkan** [reply]
I'd like to have options on an item. Let's say I'm using the dropdown to display a list of items in a dropdown
- When clicking/selecting the item, I navigate to its page
- I also have sub-options to delete/edit the item

**Reply by benoitgrelard** [reply]
The only accessible way to do this would be to have the former _"When clicking/selecting the item, I navigate to its page"_ as an item in the submenu. This isn't something you will ever see in native OS menu either for the same reasons, otherwise you are reserving some parts only for mouse or keyboard users which makes the component inaccessible.

So for your use case, the solution is:

- item 1
- item 2
  - visit
  - edit
  - delete
- item 3
